### DISCUSSION TITLE
EGA header how to ??

### DISCUSSION THREAD

**Discussion by oO-cozy-Oo** [UzixLS/zx-sizif-512#17] [General]
Just trying to get a better video out put, as all I`ve go is HDMI and VGA monitors, and the ega header seems the best option.
I dont want to use a RGB to HDMI convertor

**Comment by UzixLS** [comment]
Never tried, but I know some zx-pk.ru users have successfully connected zxkit1 vga converter to the EGA header.
Also note that EGA have 6-bit color, so ULA+ screen (which have 8-bit color) will not display correctly.
